app/assets/javascripts/tictactoe.js

```diff
@@ -1 +1,125 @@
-// Code your JavaScript / jQuery solution here
+var board = []
+var turn = 0
```

Comment:
Prefer `let` or `const` over var -- see here for more info:  https://hackernoon.com/js-var-let-or-const-67e51dbb716f